metrics.md

```diff
@@ -0,0 +1,85 @@
+# Universal Search Metrics
```

Comment:
nope, still draft as far as I know.  The discussion for Fx is happening in https://bugzilla.mozilla.org/show_bug.cgi?id=1240849